core_impl/src/test/java/com/google/instrumentation/stats/StatsManagerImplTest.java

```diff
@@ -54,7 +56,7 @@
 
 
   private final StatsManagerImplBase statsManager =
-      new StatsManagerImplBase(new SimpleEventQueue());
+      new StatsManagerImplBase(new SimpleEventQueue(), TestClock.create(Timestamp.create(1, 2)));
```

Comment:
Could we store a reference of this TestClock and compare the actual start and end time of views against it for the tests below? (line 101)